requirements/base.txt

```diff
@@ -15,3 +15,4 @@ django-allauth==0.51.0  # https://github.com/pennersr/django-allauth
 django-crispy-forms==1.14.0  # https://github.com/django-crispy-forms/django-crispy-forms
 crispy-bootstrap5==0.6  # https://github.com/django-crispy-forms/crispy-bootstrap5
 django-redis==5.2.0  # https://github.com/jazzband/django-redis
+cloudipsp
```

Comment:
No version, source link

Reply:
This package is partly broken. See cloudipsp/python-sdk#5